Description: Do not depend on libtest_mimic
 This patch removes the dev-dependency on libtest_mimic v0.4: the
 library's API changed substantially in v0.5 and Debian currently
 ships v0.7.
Author: NoisyCoil <noisycoil@tutanota.com>
Forwarded: not-needed
Last-Update: 2025-03-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -79,10 +79,10 @@
 name = "test_round_trip"
 path = "tests/test_round_trip.rs"
 
-[[test]]
-name = "yaml-test-suite"
-path = "tests/yaml-test-suite.rs"
-harness = false
+#[[test]]
+#name = "yaml-test-suite"
+#path = "tests/yaml-test-suite.rs"
+#harness = false
 
 [dependencies.arraydeque]
 version = "0.5.1"
@@ -94,8 +94,8 @@
 [dependencies.hashlink]
 version = "0.10"
 
-[dev-dependencies.libtest-mimic]
-version = "0.4.0"
+#[dev-dependencies.libtest-mimic]
+#version = "0.4.0"
 
 [dev-dependencies.quickcheck]
 version = "1.0"
